### ISSUE TITLE
Round 2 started

### ISSUE THREAD

**Issue by drdhaval2785** [drdhaval2785/SanskritVerb#606]
Once the whole exercise was over. Necessary corrections were made in the code.
These may have cause erstwhile correct forms to fail.
Therefore, trying a cursory testing named Round 2.
Will release the code at the end of round 1 of testing as a release version.
This is just for documentation.

**Comment by drdhaval2785** [comment]
For round 2, I have made some customization.
1. The verb list which earlier had verbs conjugated by Gerard, has now been changed to the verbs where SK shows application of a new verb (to make verb list representative).
2. Earlier the verb list was around 450. Now it has been pruned to 181.

The statistics are also worth noting.
Before - The suspect verb forms were 11650 for 450 verbs.
After - The suspect verb forms are 926 for 181 verbs.

So, a lot of improvement in output has happened. Still some more to go.
